src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -418,13 +419,14 @@ void disownRecovery(String stairwayId) throws DatabaseOperationException, Interr
         NamedParameterPreparedStatement statement =
             new NamedParameterPreparedStatement(connection, sql)) {
 
-      startReadOnlyTransaction(connection);
+      startTransaction(connection);
```

Comment:
Would you kindly comment on why this was necessary?

Reply:
My concern, may be unfounded, was that the snapshot transaction would be reading out of date data because there would be too many other transactions writing to the table. This change makes sure that the read transaction is ordered with the write transactions. I will add a comment.